src/utils/DatabaseFilter.cpp

```diff
@@ -56,7 +56,7 @@ using namespace std;
 
     Implemented filter criteria:
 
-        accession: Filter database according to the set of protein accessions contained in an identification file (idXML, mzIdentML)
+        ID: Filter database according to the set of proteinIDs contained in an identification file (idXML, mzIdentML)
```

Comment:
I would keep "protein accession" as we don't filter according to protein_identifications but the protein accessions contained in the peptide_identifications